cmd/cmount/mount_brew.go

```diff
@@ -0,0 +1,32 @@
+// Build for macos with the brew tag to handle the absence
+// of fuse and print an appropriate error message
+
+// +build brew
```

Comment:
Does this need to mention the cmount tag? Or do we do this regardless of the cmount tag?

If we can expect `cmount` and `brew` to be set then we need to adjust the constraints for `cmount/{fs,mount}.go` something like:

```
// +build cmount
// +build cgo
// +build linux darwin,!brew freebsd windows
```

Reply:
I will just be replacing the `cmount` tag with the `brew` tag in Homebrew so this case probably won't happen there but I can add it anyway to be safe if you want?

Reply:
No I think that is fine. Adding too many build constraints makes my head explode anyway as there are too many combinations to think through.